libraries/support/azure/service.rb

```diff
@@ -52,12 +66,16 @@ def set_reader(name, value, override)
       self
     end
 
-    def get(url:, api_version:)
+    def get(url:, api_version:, error: nil, unwrap: nil)
       confirm_configured!
 
       cache.fetch(url) do
         body = rest_client.get(url, params: { 'api-version' => api_version }).body
-        body.fetch('value', body)
+
+        error&.call(body)
```

Comment:
You know, I've disregarded the shorthand for `call` but `error&.(body)` seems much more to the point than `error&.call(body)`. We don't have to change anything here, but food for thought.